src/app/pages/dashboard/dashboard.module.ts

```diff
@@ -40,11 +41,12 @@ import * as echarts from 'echarts';
     DashboardComponent,
     JobResultComponent,
     JobActionsMenuComponent,
-    ViewResultButtonComponent,
+    ViewJobAnalyzersResultButtonComponent,
```

Comment:
The renaming is pretty redundant now so I think it's best to revert it back to it's original name.